docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
```

Comment:
Do you really have any env your project depend on (I mean except of db of course)